gbmodel/model_sqlalchemy.py

```diff
@@ -448,6 +448,22 @@ def insert_session(self, term, year, professor_id):
         db.session.commit()
         return id
 
+    # Removes an entire session with all the teams and students
+    # Input: session id
+    def remove_session(self, session_id):
```

Comment:
if you have time, it'd be nice to have this moved to docstrings rather than comments